samples/embedded-login/src/main.js

```diff
@@ -13,14 +13,14 @@ import * as forgerock from '@forgerock/javascript-sdk';
 const FATAL = 'Fatal';
 forgerock.Config.set({
   clientId: process.env.WEB_OAUTH_CLIENT, // e.g. 'ForgeRockSDKClient'
-  redirectUri: `${window.location.origin}/callback`, // e.g. 'https://sdkapp.example.com:8443/_callback'
-  scope: process.env.SCOPE, // e.g. 'openid profile me.read'
+  redirectUri: `${window.location.origin}/callback`, // e.g. 'https://sdkapp.example.com:8443/callback'
+  scope: process.env.SCOPE, // e.g. 'openid profile email address phone me.read'
   serverConfig: {
-    baseUrl: process.env.AM_URL, // e.g. 'https://openam.example.com:9443/openam/'
+    baseUrl: process.env.AM_URL, // e.g. 'https://myorg.forgeblocks.com/am' or 'https://openam.example.com:8443/openam/'
```

Comment:
The first example URL is /am and the second is /openam/ - I don't think it matters to the samples whether there is one or not, but better to be consistent?